contracts/staking/SimpleStaking.sol

```diff
@@ -126,7 +124,7 @@ contract SimpleStaking is AbstractGoodStaking, StakingToken {
 
 	function withdrawRewards() public {
 		FundManager fm = FundManager(nameService.getAddress("FUND_MANAGER"));
-		fm.mintReward(address(iToken), msg.sender); // send rewards to user
+		fm.mintReward(nameService.getAddress("CDAI"), msg.sender); // send rewards to user and use cDAI address since reserve in cDAI
```

Comment:
generic

Reply:
Yes but reserve just in cDAI and this parameter needed to update RR so must be cDAI